book/src/theme-gallery/index.md

```diff
@@ -106,3 +107,28 @@ It features a dominant partition of space into a bright and a dark side.
 ```
 ![bipartite theme screenshot](./bipartite.png)
 
+## University
+_[go to top](#theme-gallery)_
+
+This theme is a simple and configurable theme.
+
+### Options
+- `university-name`: The University name
```

Comment:
Thinking about it, we should maybe call this `institution-name` right? The theme can be called whatever, but a more general parameter name here could make this theme more broadly "applicable".

Reply:
Definitely better. Fixed.